LUIS/webpack.config.js

```diff
@@ -0,0 +1,43 @@
+// This is a template webpack config file with minimal configuration.
+// Users are free to create their own webpack configuration files in their application.
+const webpack = require('webpack');
+const path = require('path');
+module.exports = {
+  entry: './lib/luisAuthoring.ts',
+  devtool: 'source-map',
+  output: {
+    filename: 'luisAuthoringBundle.js',
+    path: __dirname,
+    libraryTarget: 'var',
```

Comment:
If the target is `umd`, tree-shaking is still possible by most bundlers. I there a technical reason this is bundled as "all or nothing"?